src/xdocs/config_naming.xml

```diff
@@ -1145,7 +1145,25 @@ class MyClass {
       <subsection name="Description">
         <p>
           Checks that method and <code>catch</code> parameter names conform to a format specified
-          by the format property.
+          by the format property. By using <code>scope</code> and <code>excludeScope</code> properties
+          it is possible to specify different formats for methods at different visibility levels.
+          For example, the following configuration checks that the parameters always start with two
+          lowercase characters and, in addition, that public method parameters cannot be one character
+          long:
+          <pre>
+&lt;module name=&quot;ParameterName&quot;&gt;
+    &lt;property name=&quot;format&quot; value=&quot;^[a-z]([a-z0-9][a-zA-Z0-9]*)?$&quot;/&gt;
+    &lt;property name=&quot;excludeScope&quot; value=&quot;public&quot;/&gt;
+    &lt;message key=&quot;name.invalidPattern&quot;
+        value=&quot;Parameter name ''{0}'' must match pattern ''{1}''.&quot;/&gt;
+&lt;/module&gt;
+&lt;module name=&quot;ParameterName&quot;&gt;
+&lt;property name=&quot;format&quot; value=&quot;^[a-z][a-z0-9][a-zA-Z0-9]*$&quot;/&gt;
+    &lt;property name=&quot;scope&quot; value=&quot;public&quot;/&gt;
+    &lt;message key=&quot;name.invalidPattern&quot;
+        value=&quot;Parameter name ''{0}'' must match pattern ''{1}''./&gt;
```

Comment:
Example of validation not being run against this example. Line is missing end quote to value attribute.